vgaidarji/react-graphql-playground
# Increase code coverage on the project

### What has been done
1. Covered remaining missed statements/branches with tests.

### How to test
1. Run `npm run tests-with-coverage` locally or check CI build logs and confirm that tests succeed and coverage improved (ideally 100% as of this PR, may decrease in future).
